liza/number_check.rb

```diff
@@ -0,0 +1,22 @@
+def number_check(num)
+  result = if qualify?(num)
+             'The number is qualify.'
+           else
+             'The number is not qualify.'
```

Comment:
The number qualifies
The number does not qualify

Reply:
Done